pkg/ytconfig/logging.go

```diff
@@ -112,14 +112,27 @@ func newLoggingBuilder(location *ytv1.LocationSpec, componentName string) loggin
 	}
 }
 
-func newJobProxyLoggingBuilder() loggingBuilder {
-	return loggingBuilder{
+type jobProxyLoggingBuilder struct {
+	loggingDirectory string
+	componentName    string
+	// COMPAT(ignat)
+	logging            Logging
+	logManagerTemplate Logging
```

Comment:
You are adding some configuration for `logManagerTemplate`, but I don't see usage of this field. Only `logging` field is passed to the config generator, so resulting config will be the same as before. 
Am I missing something?